community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/StageControl.java

```diff
@@ -26,4 +26,6 @@
 public interface StageControl
 {
     void panic( Throwable cause );
```

Comment:
We discussed it, but opted out because Panicable instances report and receive panic whereas  StageControl spreads panic, managing the Panicables, sort of. To have even StageControl be Panicable would make code navigation hard because you'd find yourself navigating around in loops without knowing where to really go.